src/views/Migrate.vue

```diff
@@ -10,33 +18,56 @@ import { Component, Vue } from 'vue-property-decorator';
 import { AccountInfo } from '@/lib/AccountInfo';
 import { WalletStore } from '@/lib/WalletStore';
 import { WalletInfo, WalletType } from '@/lib/WalletInfo';
+import { SmallPage } from '@nimiq/vue-components';
+import Loader from '@/components/Loader.vue';
+import { KeyguardClient } from '@nimiq/keyguard-client';
 
-@Component
+@Component({components: {SmallPage, Loader}})
 export default class Migrate extends Vue {
-
-    // TODO: Move to CONSTANTS
-    private static readonly RETURN_WAIT_TIME: number = 2000;
+    private title: string = 'Migrating your accounts';
     private status: string = 'Connecting to Keyguard...';
+    private state: Loader.State = Loader.State.LOADING;
+    private message: string = '';
+
+    private static: {keyguardClient?: KeyguardClient} = {};
+
+    public mounted() {
+        this.static.keyguardClient = this.$rpc.createKeyguardClient();
+
+        // Need to freeze the object to prevent Vue's reactivity system
+        // from accessing properties of the cross-origin `window.opener`,
+        // which throws security errors in browsers.
```

Comment:
Why not actually declare the keyguardClient as static property of the class to avoid this problem?
Naming it `_keyguardClient` might also work.

Reply:
maybe even 
```
private static: {keyguardClient?: KeyguardClient};
``` 
does the trick. If i recall correctly, uninitialised properties will not be made reactive by vue.

Reply:
@nibhar You, sir, are a genius! :nerd_face: